components/subcomponents/movieNight/suggestionDatabase.py

```diff
@@ -0,0 +1,167 @@
+        self.shelve.close()
+
+        return movie
+
+    # Allows up to MAX_SUGGESTIONS suggestions to be stored per user
+    def add_suggestion(self, member: str, suggested_movie: Movie) -> discord.Embed:
+        reply = self.__embed_movie(suggested_movie)
+
+        if self.has_space(member):
+            # Update Database if there is space
+            db = self.shelve.open()
+            if db.get(member) is None:
+                db[member] = [suggested_movie]
+            else:
+                db[member] = db[member] + [suggested_movie]
+            self.shelve.close(modified_dict=db)
+
+            reply.title = f"Movie successfully added to {member}'s list!"
+        else:
+            # Send error if suggestion list is full
+            reply.color = ut.embed_colour["ERROR"]
+            reply.title = "Suggestion List at Capacity!"
+            reply.description = "Please remove some suggestions before adding new ones."
+
+        return reply
+
+    def remove_suggestion(self, member: str, movie_name: str) -> discord.Embed:
+        reply = discord.Embed(colour=ut.embed_colour["MOVIE_NIGHT"])
+
+        movie = self.get_movie(member, movie_name)
+        if movie:
+            db = self.shelve.open()
+            new_suggestion_list = db.get(member)
+            new_suggestion_list.remove(movie)
+            db[member] = new_suggestion_list
+            self.shelve.close(modified_dict=db)
+
+            reply = self.__embed_movie(movie)
+            reply.title = f"Movie Successfully Removed from {member}'s list !"
+        else:
+            reply.colour = ut.embed_colour["ERROR"]
+            reply.title = "Removal Unsuccessful!"
+            reply.description = f"The movie `{movie_name}` was not found in {ut.get_member_str(member)}'s suggestion list."
+
+        return reply
+
+    def get_suggestion_names(self, member: str) -> List[str]:
+        db = self.shelve.open()
+
+        suggested_movies_names = []
+        suggested_movies = db.get(member)
+        if suggested_movies:
+            suggested_movies_names = [movie.name for movie in suggested_movies]
+        self.shelve.close()
+
+        return suggested_movies_names
+
+    def get_list_embed(self) -> discord.Embed:
+        reply = discord.Embed(colour=ut.embed_colour["MOVIE_NIGHT"])
+
+        members = self.get_members()
+        if members:
+            reply.title = "Suggestion List"
+            reply.description = ""
+            for member in members:
+                # Format list
+                suggested_names_str = ", ".join(self.get_suggestion_names(member))
+                reply.description += f"**{ut.get_member_str(member)}**: {suggested_names_str}\n"
+        else:
+            reply.colour = ut.embed_colour["ERROR"]
+            reply.title = "Movies not found!"
+            reply.description = "Everyone's suggestion list is empty."
+
+        return reply
+
+    def get_suggestion_embed(self, member: str, movie_name: str) -> discord.Embed:
```

Comment:
can u leave some small comments above these methods

Reply:
Added to the backlog 💀